docs/document.rst

```diff
@@ -1241,6 +1244,28 @@ For details on **embedded files** refer to Appendix 3.
 
        2. *docsrc* TOC entries **will not be copied**. It is easy however, to recover a table of contents for the resulting document. Look at the examples below and at program `PDFjoiner.py <https://github.com/pymupdf/PyMuPDF-Utilities/tree/master/examples/PDFjoiner.py>`_ in the *examples* directory: it can join PDF documents and at the same time piece together respective parts of the tables of contents.
 
+
+    .. index::
```

Comment:
I'm not really sure what `..index` does - is it for better search indexing or something?

Reply:
Hmm, i don't know either. I think i was hoping you're be able to tell me!

I think there are quite a few other examples of the same `.. index::` pattern elsewhere in docs/document.rst.